app/src/main/res/values/strings.xml

```diff
@@ -55,8 +55,17 @@
     <string name="watomatic_github_url" translatable="false">https://github.com/adeekshith/watomatic</string>
     <string name="watomatic_wato_message_url" translatable="false">https://www.reddit.com/r/watomatic/comments/lrkn7j/fellow_watos_share_your_creative_wato_message/</string>
 
+    <!-- Share debug logs -->
+    <string name="app_logs_file_name">WatomaticLogs.txt</string>
+    <string name="share_email_body">Hi Team Watomatic,\nPlease find the attachment of the app logs.</string>
+    <string name="share_email_subject">Watomatic AppLogs</string>
+    <string name="share_email_id">watomatic@deekshith.in</string>
+    <string name="share_email_err_msg">Couldnot find any application to process this request</string>
+
     <!-- keys and app internal strings -->
     <string name="key_pref_app_language" translatable="false">pref_app_language</string>
+    <string name="pref_show_notification" translatable="false">pref_show_notification</string>
```

Comment:
`pref_show_notification` -> `pref_show_notification_replied_msg` for value to future proof (but optional)